app/src/main/java/com/better/alarm/presenter/AlarmsListActivity.kt

```diff
@@ -189,13 +203,52 @@ class AlarmsListActivity : AppCompatActivity() {
     override fun onStart() {
         super.onStart()
         configureTransactions()
+        configureSnackbar()
+    }
+
+    private fun configureSnackbar() {
+        val rootView = window.decorView.rootView
```

Comment:
Can this fail? Does it make sense to move it inside the subscribe block?

Reply:
Yeah it does makes sense.